runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -180,6 +180,10 @@
 	private LoadingCache<NamedQuantity, Boolean> highlightedItems;
 	private LoadingCache<NamedQuantity, Boolean> hiddenItems;
 	private final Queue<Integer> droppedItemQueue = EvictingQueue.create(16); // recently dropped items
+	private final Queue<Integer> tableItemQueue = EvictingQueue.create(16); // recently placed items
```

Comment:
I guess you are just copying the dropped item queue logic here; but the dropped item queue exists because you can drop multiple items on each tick, and it needs to be able to track each of them. However you cannot place multiple items on a table on the same tick since it is an object interaction, and so I think this is unnecessary.